extensions/search/schema.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
I honestly have no idea if this is the right way to express this in JSON Schema

Reply:
I can take over the JSON Schema and fix it if you like me to do that.

Reply:
I added you as a collaborator, or you can just send me the corrected schema.

Reply:
Thanks, updated.